plugin-gradle/src/main/java/com/diffplug/gradle/spotless/SpotlessPlugin.java

```diff
@@ -26,13 +26,14 @@
 
 public class SpotlessPlugin implements Plugin<Project> {
 	static final String SPOTLESS_MODERN = "spotlessModern";
-	static final String MINIMUM_GRADLE = "6.1.1";
+	static final String VER_GRADLE_min = "6.1.1";
+	static final String VER_GRADLE_javaPluginExtension = "7.1";
```

Comment:
May need to bump `VER_GRADLE_min` to `7.1`, using conventions will start emitting deprecation warnings in Gradle 8.1, and will be removed in the future Gradle 8 versions.

Reply:
Roger. If we are throwing warnings that we can't easily workaround with a version check, then I'm happy to bump the minimum.

Reply:
@nedtwigg Are you planning to go on Gradle plugin 7? If so can we bump the ming Gradle requirement to 7.1 for something like this?

Reply:
And 682dcd6d4840002adb834b2db67038869efe5d57.